prow/cmd/peribolos/main.go

```diff
@@ -714,15 +714,15 @@ func configureOrg(opt options, client *github.Client, orgName string, orgConfig
 	}
 
 	for name, team := range orgConfig.Teams {
-		err := configureTeamAndMembers(client, opt.fixTeamMembers, githubTeams, name, orgName, team, invitees, nil)
+		err := configureTeamAndMembers(opt, client, githubTeams, name, orgName, team, nil)
 		if err != nil {
 			return fmt.Errorf("failed to configure %s teams: %v", orgName, err)
 		}
 	}
 	return nil
 }
 
-func configureTeamAndMembers(client *github.Client, fixMembers bool, githubTeams map[string]github.Team, name, orgName string, team org.Team, invitees sets.String, parent *int) error {
+func configureTeamAndMembers(opt options, client *github.Client, githubTeams map[string]github.Team, name, orgName string, team org.Team, parent *int) error {
```

Comment:
Why pass in `opt` instead of `fixTeamMembers`?

Reply:
I left this for readability, mostly. When I was working on this, I had copied the entire `orgInvitations` function which had some opt checks.. those checks were redundant this far into the fix-members process, so I stripped them.. but this just makes the entire set of options available, if we in the future need to check an option further into the process.

It's possible to revert to only passing the bool in, but IMO this just makes it cleaner.